composer.json

```
@@ -86,7 +87,20 @@
             "dev-master" : "4.0.x-dev"
         }
     },
-    "config" : {
-        "bin-dir" : "bin"
+    "scripts": {
+        "phpstan": [
+            "@php vendor/bin/phpstan analyse lib tests"
+        ],
+        "php-cs-fixer": [
```

I wouldnt name the fixer so we can change the tool beeing used without the need to chnage the devs workflow commands 
```suggestion
        "csfix": [
```

---

Will do.